ONSdigital/census-rh-ui
# Cr 1140 authentication timeout

# Motivation and Context
Change so that a user session timeout in RHUI always results in a timeout page, rather than an error reload of the UAC authentication page

# What has changed
Change to the HTTPForbidden resolution to render the standalone timeout page, rather than rendering the UAC authentication page
Change to remove flash message update for timeout
Restructure of direct access unit tests to new methods, and merging of existing tests to three super tests
No RHCucumber updates required

# How to test?
In RHUI, UAC authenticate, then allow session timeout or remove browser session cookie on various pages in the journey (before reaching EQ) and confirm that the dedicated timeout page is rendered, and that when reaching the UAC authentication page it does not display any in page error messages

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1140
